impeller/renderer/backend/metal/context_mtl.mm

```diff
@@ -88,24 +88,6 @@ static bool DeviceSupportsComputeSubgroups(id<MTLDevice> device) {
   sync_switch_observer_.reset(new SyncSwitchObserver(*this));
   is_gpu_disabled_sync_switch_->AddObserver(sync_switch_observer_.get());
 
-  // Worker task runner.
-  {
-    raster_message_loop_ = fml::ConcurrentMessageLoop::Create(
-        std::min(4u, std::thread::hardware_concurrency()));
-    raster_message_loop_->PostTaskToAllWorkers([]() {
-      // See https://github.com/flutter/flutter/issues/65752
-      // Intentionally opt out of QoS for raster task workloads.
-      [[NSThread currentThread] setThreadPriority:1.0];
```

Comment:
Get rid of the corresponding Foundation.h header import above?

Reply:
Done